terragrunt-config/environments/prod/eks_autoscaler/terragrunt.hcl

```diff
@@ -0,0 +1,13 @@
+/*BC: the following functions help to access parent folders, to get to the main terraform files*/
+include {
+  path = find_in_parent_folders()
+}
+ /* BC: In this line you can reference either the direct folder of your terraform files, or github submodules. */
+terraform {
+  source = "../../src/3-eks-cluster-autoscaler" 
```

Comment:
The original module source was a git source...why was it opted to use a local path here?

https://github.com/ManagedKube/kubernetes-ops/blob/main/terraform-environments/aws/dev/25-eks-cluster-autoscaler/main.tf#L70

Reply:
The reason is that for testing the entire infrastructure in my repo, it was easier to do it by reference, for my quick testing and understanding of the entire solution, however I consider it appropriate for a formal environment to use github modules. Do you need me to change it at once?

Reply:
Hi @sekka1 , I agree with you, the benefits of using GitHub modules as references, are much better, since you can use the version; personally I am not very familiar with githubmodules, but I can try to implement it for tomorrow. Do you agree?

Reply:
This is similar to any other terraform module.  Like if you were to use an official AWS module like the VPC or EC2...you wouldnt go and pull down the source code for that to develop locally.  You would just reference it.

Reply:
got it, it is handled as a provider. i will be checking